modules/sdk/gradle-plugins-workspace/src/gradleTest/workspaceProdctInfoBadProductKey/build.gradle

```
@@ -0,0 +1,15 @@
+task runGradleTest
+
+runGradleTest {
+	doLast {
+		String tomcatVersion = null
+
+		try {
+			tomcatVersion = gradle.liferayWorkspace.appServerTomcatVersion
+		}
+		catch (Exception exception) {
+		}
+
+		assert version == null
```

```suggestion
		assert tomcatVersion == null
```

Should this be tomcatVersion?